presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -385,6 +386,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Experimental: enable pushdown of range filters on subscripts (a[2] = 5) into ORC column readers",
                         hiveClientConfig.isRangeFiltersOnSubscriptsEnabled(),
                         false),
+                booleanProperty(
+                        ADAPTIVE_FILTER_REORDERING_ENABLED,
+                        "Experimental: enable adaptive filter reordering",
```

Comment:
Dumb question . If it is an `experimental` property shouldn't the default value be false always.

Reply:
@ajaygeorge This property exists only within the context of filter pushdown. Since filter pushdown is "experimental", it follows that this property is experimental as well. The default false is "true", because this is essentially a kill switch that I'd like to have just in case. I expect it to be used very rarely (if ever).